### PR TITLE
Changes (S3)

### PR DESCRIPTION
**Pull request by pamichals**

### CODE REVIEW

**Review by claesmk — CHANGES_REQUESTED** [review]
It looks like you have removed all of the other code and replaced it with yours.  You need to merge your specific handler code into upstream, not replace it

**Comment by pamichals** [comment]
I added all of the removed code and pushed back to the origin. Sorry!

File: log_parse.py [hunk 1]
```diff
@@ -23,26 +23,24 @@ def handle_message(self, df, id):
         elif id == 114001:
             # %ASA-1-114001: Failed to initialize 4GE SSM I/O card (error error_string).
             m = re.search(r'card \(error (\w+)\)', df.loc[id, 'Text'])
-            if m:
-                df.loc[id, 'Error'] = m.group(1)
+        elif id == 114009:
```
Comment (claesmk) [inline log_parse.py:26]: Here is an example of where it looks like you have merged your branch incorrectly with upstream.  I'm not entirely sure how this happened.  If you look at [this](https://github.com/NCATComp410/comp410_summer21/blob/4ab66d1e8c16adf13a5e2dd5394e33875f08e665/log_parse.py#L23) line of code, you'll see how it should look.  You shouldn't be removing the if m: clause and the following line

**Comment by claesmk** [comment]
It looks like your merge is still incorrect for some reason.  Maybe you made a mistake when resolving conflicts?  Which tool did you use for that? I have inserted an example of the incorrect merge above in a review comment format

**Comment by pamichals** [comment]
I have a Mac, so I'm using TextEdit (default program) to make my edits. I added the if m portion back and now I just have an AssertionError on my specific portion of the assignment

Screen Shot 2021-07-14 at 8 18 03 PM

**Comment by claesmk** [comment]
if you look at your assigned message in [syslogs.txt ](https://github.com/NCATComp410/comp410_summer21/blob/4ab66d1e8c16adf13a5e2dd5394e33875f08e665/syslogs.txt#L261) - your message string does not match the one you currently have in your unit-test.  The correct one is 
```
        self.assertTrue(self.df.loc[114009, 'Text'] == 'Failed to set multicast address in 4GE SSM I/O card (error error_string).')
```

So the message string doesn't match which causes the assertion to fail.  If you correct the message is should work OK

**Comment by pamichals** [comment]
Thank you for the help! Adding a teammate for review shortly

Have a great night!

**Review by claesmk — APPROVED** [review]
I'll add my review as well.  You can go ahead and merge your pull request.

**Pull request merged**